internal/jobcontainers/mounts.go

```diff
@@ -0,0 +1,56 @@
+package jobcontainers
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+)
+
+// namedPipePath returns true if the given path is to a named pipe.
+func namedPipePath(p string) bool {
```

Comment:
nit: imo I prefer for this to have some sort of verb at the beginning, like "isNamedPipePath"

Reply:
Sure